week-2/G-conditionals-2/exercise-1.js

```diff
@@ -7,9 +7,14 @@
 */
 
 function negativeOrPositive(number) {
-
+if(number< 0) {
+  return "negative";
+}else if(number >= 0) {
```

Comment:
Once you are in the "else" block, that means the if statement in line 10 already evaluated to false. So you don't need to check again here. Just say `} else {`